{{ cookiecutter.project_slug }}/src/{{ cookiecutter.project_slug }}/core/templates/core/login.html

```diff
@@ -0,0 +1,22 @@
+{% extends "admin/login.html" %}
+{% load i18n %}
+{% block content %}
+<div id="content-main">
+
+{% if user.is_authenticated %}
+<p class="errornote">
+{% blocktranslate trimmed %}
+    You are authenticated as {{ username }}, but are not authorized to
+    access this page. Would you like to login to a different account?
+{% endblocktranslate %}
+</p>
+{% endif %}
+
+<form id="login-form" method="get" action="{% url 'simple_openid_connect:login' %}">
+  <div class="submit-row">
+    <input type="submit" value="{% translate 'Log in with Mafiasi' %}">
+  </div>
+</form>
```

Comment:
Any reason, this is a form instead of
```html
<a href="...">
    <button>Log in with Mafiasi</button>
</a>
```

Reply:
I wanted to keep all of django's CSS instead of replicating the design with custom CSS.

Reply:
Huh alright. Normally I don't like this since it's really against HTML semantics and not very screen-reader friendly but since this site is only intended for us admins, I guess it's fine.